tests/auth_tests/test_auth_backends.py

```diff
@@ -583,6 +583,11 @@ def authenticate(self, request, username=None, password=None):
         raise TypeError
 
 
+class NoAuthenicateArgumentsBackend:
```

Comment:
Sorry just browsing here but there's a typo : `NoAuthenicateArgumentsTest` instead of `NoAuthenticateArgumentsTest`